src/sonic_ax_impl/mibs/vendor/cisco/ciscoSwitchQosMIB.py

```diff
@@ -62,8 +62,6 @@ def __init__(self):
         self.mib_oid_to_queue_map = {}
         self.mib_oid_list = []
 
-        self.if_range = []
-
         self.reinit_data()
```

Comment:
>reinit_data [](start = 13, length = 11)

As a design principal for updater classes, let's keep the __init__() as simple as possible, so there is no exception at all. Leave all the complex logic in reinit_data() and update_data(), where any exception will be handled well. If any zero initialization needed, please added it in __init__().

Reply:
@qiluo-msft 
The design of MIB is aligned with current MIBs like rfc1213.py or rfc2863.py which also use update or reinit in __init__. 
The reason to use reinit_data() or update_data() is that MIB should be initialized immediately. It's also required by unit tests. If MIB isn't initialized immediately, the unit tests will fail because testing data won't be available.

Reply:
We recently changed the design, please check [https://github.com/Azure/sonic-snmpagent/commit/5580935de8f82caae071ef62cf9dc647b8f78ca0](https://github.com/Azure/sonic-snmpagent/commit/5580935de8f82caae071ef62cf9dc647b8f78ca0).
Your reason is solvable, and you may find similar cases in the same commit.

---
In reply to: [158642404](https://github.com/andrii-savka/sonic-snmpagent/pull/5#discussion_r158642404) [](ancestors = 158642404)

Reply:
ok. Thanks for the reference. 
I will align the MIB with the updated MIBUpdater design.

Reply:
Done.